ext/random/tests/02_engine/all_serialize_error.phpt

```diff
@@ -3,63 +3,65 @@ Random: Engine: serialize error pattern
 --FILE--
 <?php
 
-echo \Random\Engine\Mt19937::class . PHP_EOL;
+echo \Random\Engine\Mt19937::class, PHP_EOL;
```

Comment:
nits: It may be a good idea to use `use`.

Reply:
Not sure here! The engine is not actually used, this is just for "pretty printing".